protocol/contracts/interfaces/v2/strategies/ITempleStrategy.sol

```diff
@@ -94,12 +94,19 @@ interface ITempleStrategy is ITempleElevatedAccess {
     function currentDebt() external view returns (uint256);
 
     /**
-     * @notice How much this strategy is free to borrow from the Treasury Reserves Vault
-     * @dev This is bound by:
-     *   1/ How much stables is globally available (in the TRV + in the TRV base strategy)
-     *   2/ The amount this individual strategy is whitelisted to borrow.
-     */
-    function availableToBorrow() external view returns (uint256);
+     * @notice A strategy's current amount borrowed from the TRV, and how much remaining is free to borrow
```

Comment:
I think you need to `git pull origin temple-v2`?

Reply:
Did you do this merge?

Reply:
Had an issue on the merge. Will fix it later.